Include sys/types.h before including sys/random.h to avoid:

/usr/include/sys/random.h:37:32: error: unknown type name 'u_int'
--- src/playlist.c.orig	2020-01-29 14:40:14.000000000 -0600
+++ src/playlist.c	2020-03-20 00:37:28.000000000 -0500
@@ -19,6 +19,9 @@
 #endif
 
 #include <sys/stat.h>
+#ifdef HAVE_SYS_TYPES_H
+# include <sys/types.h>
+#endif
 #ifdef HAVE_SYS_RANDOM_H
 # include <sys/random.h>
 #endif
